src/components/Abstract/Abstract.tsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/prop-types */
+
+import { Box, Container, Stack, Typography } from '@mui/material';
+import type { CommonProps, Theme } from 'types/components';
+
+export interface AbstractProps extends CommonProps {
+  overline: string;
+  title: string;
+  description: string | JSX.Element;
+  background?: string;
+  layout?: 'left' | 'center' | 'right';
+}
+
+const getBackgroundSx = (theme: Theme, background?: string) => {
+  if (!background) {
+    return {};
+  }
+
+  const overlay =
+    theme === 'dark'
+      ? 'linear-gradient(76.77deg, rgba(0, 115, 230, 0.7) 40.28%, rgba(0, 115, 230, 0) 100%), '
+      : 'linear-gradient(0deg, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0.8)), ';
+
+  return {
+    backgroundImage: `${overlay}url(${background ?? ''})`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center',
+  };
+};
+
+export const Abstract: React.FC<AbstractProps> = ({
```

Comment:
Is it ok to use React.FC @vito80ba @stratoivandiluccio? It is generally being discouraged for various reasons, you can check [this](https://github.com/facebook/create-react-app/pull/8177) for more info